pkg/collector/corechecks/servicediscovery/model/model.go

```diff
@@ -0,0 +1,20 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+// Package model contains types for service discovery.
+package model
+
+// Listener represents a listening process.
+type Listener struct {
+	PID     int      `json:"pid"`
+	Name    string   `json:"string"`
+	Environ []string `json:"environ"`
+	CWD     string   `json:"cwd"`
```

Comment:
An improvement - 
Let's treat `/` as the default value, and if it is, then we should not explicitly set it, and by that, sparing more redundant characters

Reply:
Why do we need CWD?